packages/snowflake/src/dialect.ts

```diff
@@ -61,24 +79,26 @@ export class SnowflakeDialect extends AbstractDialect {
     },
   });
 
-  readonly dataTypesDocumentationUrl =
-    'https://docs.snowflake.com/en/sql-reference/data-types.html';
-
-  // TODO: fix the minimum supported version
-  readonly defaultVersion = '5.7.0';
   readonly Query = SnowflakeQuery;
-  readonly TICK_CHAR_LEFT = '"';
-  readonly TICK_CHAR_RIGHT = '"';
   readonly connectionManager: SnowflakeConnectionManager;
   readonly queryGenerator: SnowflakeQueryGenerator;
   readonly queryInterface: SnowflakeQueryInterface;
 
-  constructor(sequelize: Sequelize) {
+  constructor(sequelize: Sequelize, options: SnowflakeDialectOptions) {
     console.warn(
       'The Snowflake dialect is experimental and usage is at your own risk. Its development is exclusively community-driven and not officially supported by the maintainers.',
     );
 
-    super(sequelize, DataTypes, 'snowflake');
+    super({
+      dataTypeOverrides: DataTypes,
+      dataTypesDocumentationUrl: 'https://docs.snowflake.com/en/sql-reference/data-types.html',
+      identifierDelimiter: '"',
+      minimumDatabaseVersion: '5.7.0',
```

Comment:
Can you add the TODO back? Since this version number was just copied over from MySQL

Reply:
done

Reply:
done!